### PR TITLE
Feature - HaNguyen - UI Comment

### PR DESCRIPTION
**Pull request by at-hanguyen3**
UI comment in detail

### CODE REVIEW

File: MyApp/View/Controller/Detail/Menu/MenuTableCellViewModel.swift [hunk 1]
```diff
@@ -20,7 +20,8 @@ final class MenuTableCellViewModel: ViewModel {
 
     // MARK: - Public
     func numberOfItems(inSection section: Int) -> Int {
-        return min(menuRestaurants.count, Configure.maxItem)
+//        return min(menuRestaurants.count, Configure.maxItem)
+        return menuRestaurants.count
```
Comment (at-hainguyen3) [inline MyApp/View/Controller/Detail/Menu/MenuTableCellViewModel.swift]: Sao phải comment đoạn này lại

File: MyApp/Model/Schema/Comment.swift [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+//  MyApp
+//
+//  Created by PCI0010 on 2/15/19.
+//  Copyright © 2019 Asian Tech Co., Ltd. All rights reserved.
+//
+
+import Foundation
+
+class Comment {
+
```
Comment (at-lamle) [inline MyApp/Model/Schema/Comment.swift]: WARNING - Lines should not have trailing whitespace.

File: MyApp/Model/Schema/Comment.swift [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+class Comment {
+    
+    // MARK: - Properties
+    var id = ""
+    var content = ""
+    var createdAt = ""
+    var userAvatar = ""
+    var username = ""
+
```
Comment (at-lamle) [inline MyApp/Model/Schema/Comment.swift]: WARNING - Lines should not have trailing whitespace.

File: MyApp/View/Controller/Detail/DetailViewModel.swift [hunk 4]
```diff
@@ -122,9 +134,34 @@ extension DetailViewModel {
...
+        static let comments: [Comment] = {
+            var commentRestaurant: [Comment] = []
+            let comment1 = Comment(id: "1",
+                                   content: "Ăn mấy lần rồi, thích nhất thố vịt quay, thố heo ăn cũng được! Nói chung sẽ ăn tiếp",
+                                   createdAt: "7/11/2018 12:14",
+                                   userAvatar: "https://kenh14cdn.com/2017/212310618041193431006884986278710988847462n-1507458620893.jpg",
+                                   username: "Quynh Như")
+            commentRestaurant.append(comment1)
+            let comment2 = Comment(id: "2",
+                                   content: "Xin thưa vs quán là tôi đặt cơm heo quay chớ ko phải cơm heo ba chỉ chiên giòn như cốm. Ăn dỡ ko tả dc. Còn cơm thì chiên bằng dầu thấy có cặn đen luôn. Lần đầu tiên ăn mà thấy bực cả mình á.",
```
Comment (at-lamle) [inline MyApp/View/Controller/Detail/DetailViewModel.swift]: WARNING - Line should be 200 characters or less: currently 238 characters

**Comment by at-lamle** [comment]
Total linter violations in pull request: 3
Serious: 0
Warnings: 3

File: MyApp/View/Controller/Detail/DetailViewModel.swift [hunk 5]
```diff
@@ -122,9 +132,34 @@ extension DetailViewModel {
...
+                                   username: "Quynh Như")
+            commentRestaurant.append(comment1)
+            let comment2 = Comment(id: "2",
+                                   content: "Xin thưa vs quán là tôi đặt cơm heo quay chớ ko phải cơm heo ba chỉ chiên giòn như cốm. Ăn dỡ ko tả dc. Còn cơm thì chiên bằng dầu thấy có cặn đen luôn.",
+                                   createdAt: "22/9/2018 12:25",
+                                   userAvatar: "https://i.ytimg.com/vi/_8SbPRKpvbI/hqdefault.jpg",
+                                   username: "Doi Nguyen")
+            commentRestaurant.append(comment2)
+            let comment3 = Comment(id: "3",
+                                   content: "Cơm thố bò ở đây ăn được, gia vị thấm đều, Vừa ăn. Cơm vịt quay ăn cũng vừa nhưng ko được ngon lắm. Nói chung cũng vừa ăn !ơm thố bò ở đây ăn được, gia vị thấm đều, Vừa ăn. Cơm vịt quay ăn cũng vừa nhưng ko được ngon lắm. Nói chung cũng vừa ăn !ơm thố bò ở đây ăn được, gia vị thấm đều, Vừa ăn. Cơm vịt quay ăn cũng vừa nhưng ko được ngon lắm. Nói chung cũng vừa ăn !",
```
Comment (at-lamle) [inline MyApp/View/Controller/Detail/DetailViewModel.swift]: WARNING - Line should be 200 characters or less: currently 414 characters

File: MyApp/View/Controller/Detail/DetailViewModel.swift [hunk 6]
```diff
@@ -122,9 +134,34 @@ extension DetailViewModel {
...
+                                   userAvatar: "https://i.ytimg.com/vi/_8SbPRKpvbI/hqdefault.jpg",
+                                   username: "Doi Nguyen")
+            commentRestaurant.append(comment2)
+            let comment3 = Comment(id: "3",
+                                   content: "Cơm thố bò ở đây ăn được, gia vị thấm đều, Vừa ăn. Cơm vịt quay ăn cũng vừa nhưng ko được ngon lắm. Nói chung cũng vừa ăn !",
+                                   createdAt: "30/10/2018 14:04",
+                                   userAvatar: "",
+                                   username: "Ho Hang Ho")
+            commentRestaurant.append(comment3)
+            return commentRestaurant
```
Comment (at-lamle) [inline MyApp/View/Controller/Detail/DetailViewModel.swift]: WARNING - Line should be 200 characters or less: currently 414 characters

**Comment by at-lamle** [comment]
Total linter violations in pull request: 2
Serious: 0
Warnings: 2

**Comment by at-lamle** [comment]
:+1: There are no linter violations.

File: MyApp/Model/Schema/Comment.swift [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+//  Comment.swift
+//  MyApp
+//
+//  Created by PCI0010 on 2/15/19.
+//  Copyright © 2019 Asian Tech Co., Ltd. All rights reserved.
+//
+
+import Foundation
+
+class Comment {
```
Comment (at-bienle) [inline MyApp/Model/Schema/Comment.swift]: Add `final`

File: MyApp/Model/Schema/Comment.swift [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+    // MARK: - Properties
+    var id = ""
+    var content = ""
+    var createdAt = ""
+    var userAvatar = ""
+    var username = ""
+    // MARK: - Init
```
Comment (at-bienle) [inline MyApp/Model/Schema/Comment.swift]: Add space before

File: MyApp/View/Controller/Detail/Comment/CommentTableViewCell.swift [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+        userNameLabel.text = viewModel.username
+        UIImageView.downloadImage(url: viewModel.thumbnailPath) { [weak self] (image) in
+            guard let this = self else { return }
+            DispatchQueue.main.async {
+                this.avatarImageView.image = image
+            }
+        }
+    }
+
+    func configUi() {
```
Comment (at-bienle) [inline MyApp/View/Controller/Detail/Comment/CommentTableViewCell.swift]: `configUI`

**Pull request merged**